pwsh/buildkimageformats.ps1

```diff
@@ -15,6 +15,10 @@ if (-Not $IsWindows) {
 
 # dependencies
 if ($IsWindows) {
+    if ($env:buildArch -eq 'Arm64') {
+        # CMake needs QT_HOST_PATH when cross-compiling
+        $env:QT_HOST_PATH = [System.IO.Path]::GetFullPath("$env:QT_ROOT_DIR\..\$((Split-Path -Path $env:QT_ROOT_DIR -Leaf) -replace '_arm64', '_64')")
```

Comment:
This feels dirty but I couldn't figure out a better way. I tried dumping all the environment variables on the runner via Write-Host but I didn't see any that point to the host Qt's folder. Maybe install-qt-action should be enhanced to set this?

Reply:
That's a good suggestion for `install-qt-action`, but this works.